core/src/main/java/org/owasp/dependencycheck/analyzer/OssIndexAnalyzer.java

```diff
@@ -337,6 +337,11 @@ private Vulnerability transform(final ComponentReport report, final ComponentRep
         // generate a reference to the vulnerability details on OSS Index
         result.addReference(REFERENCE_TYPE, source.getTitle(), source.getReference().toString());
 
+        // generate references to other references reported by OSS Index
+        for (final String externalReference : source.getExternalReferences()) {
+            result.addReference("MISC", null, externalReference);
```

Comment:
From ODC's standpoint the source of the external reference would be the OSS Index. We can't use `null` for the name as that would impact the HTML report - so the best option would like be:
```suggestion
            result.addReference("OSSIndex", externalReference, externalReference);
```